### DISCUSSION TITLE
Use mssql instead of mysql

### DISCUSSION THREAD

**Discussion by jobritz** [dani-garcia/vaultwarden#4515] [Q&A] [answered]
Hi, is it also possible to use mssql instead of mysql? Mssql is already in use at my workplace so it would prefer to use it but I'm not sure if there are any problems as it should be quite similar to Mysql if I'm correct...

**Comment by BlackDex** [comment, marked as answer]
The ORM we use does not support that. It only supports
 - PostgreSQL 
 - MySQL/MariaDB
 - SQLite
